tests/test_cache.py

```diff
@@ -0,0 +1,46 @@
+import redis
+import unittest
+from cachecontrol.caches.file_cache import FileCache
+from cachecontrol.caches.redis_cache import RedisCache
+from canonicalwebteam.http import CachedSession
+from datetime import timedelta
+from unittest import TestCase
+
+
+def create_redis_connection_pool(
+    redis_port="6379", redis_host="localhost", redis_db=0
+):
+    return redis.ConnectionPool(host=redis_host, port=redis_port, db=redis_db)
+
+
+class TestCachedSession(TestCase):
```

Comment:
```suggestion
class TestCachedSession(unittest.TestCase):
```

Reply:
Its imported directly as well. Is that a pattern that is accepted? Or should I do
```python
from unittest import unittest, TestCase
```
or just
```
import unittest
```

instead?

Reply:
Ok, the first one does not work.

Have updated to `import unittest`